docs/resources/cce_addon.md

```diff
@@ -15,6 +15,7 @@ resource "huaweicloud_cce_addon" "addon_test" {
 
 ## Argument Reference
 The following arguments are supported:
+* `region` - (Optional) The region in which to obtain the cce resource. If omitted, the provider-level region will work as default. Changing this creates a new resource.
```

Comment:
s/the cce resource/the cce addon resource/

Reply:
In this request, specific resource name is added in the description in resource docs.